src/lib/components/Homepage/EmailForm.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  import Pill from '$lib/design-components/Pill.svelte';
+  import { theme } from '$lib/theme';
+</script>
+
+<div class="rectangle">
+  <div class="update-banner">
+    <div style="padding: 0px 8px;">
+      <p class="title">Receive an Update when we&apos;ll go live!</p>
+      <p class="subtitle">News from the Scuffle Engineering Team</p>
+    </div>
+    <div class="entry-field-container">
+      <input class="entry-field" placeholder="Your email address" type="email" />
+      <Pill color={theme.colors.brown600}>
+        <p class="button-text">Subscribe</p>
+      </Pill>
+    </div>
+  </div>
+</div>
+
+<style lang="scss">
```

Comment:
I think this doesnt do anything?

Reply:
i guess you're right.. my syntax highlighting looks a little odd sometimes without it but safe to remove